Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -24,17 +28,52 @@ public final class PaymentMethodsViewModel: PaymentMethodsViewModelType,
 PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
 
   public init() {
-
     let paymentMethodsEvent = self.viewDidLoadProperty.signal
       .switchMap { _ in
         AppEnvironment.current.apiService.fetchGraphCreditCards(query: UserQueries.storedCards.query)
           .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
           .materialize()
-      }
+    }
+
+    let deletePaymentMethodEvents = self.didDeleteCreditCardSignal.switchMap { creditCard in
+      AppEnvironment.current.apiService.deletePaymentMethod(input: .init(paymentSourceId: creditCard.id))
+        .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+        .materialize()
+    }
+
+    let deletePaymentMethodEventsErrors = deletePaymentMethodEvents.errors()
+
+    self.showAlert = deletePaymentMethodEventsErrors
+      .ignoreValues()
+      .map {
+        localizedString(
+          key: "Something_went_wrong_and_we_were_unable_to_remove_your_credit_card_please_try_again",
+          //swiftlint:disable:next line_length
+          defaultValue: "Something went wrong and we were unable to remove your credit card, please try again."
```

Comment:
Copy subject to review!

Reply:
Does this needed to get updated to the new YAML key (line 50) and value (line 52) from 
https://github.com/kickstarter/kickstarter/pull/13237?